lib/Bio/EnsEMBL/DataCheck/Checks/MartDatasetName.pm

```diff
@@ -60,7 +60,6 @@ sub tests {
   my $dbname = $self->dba->dbc->dbname;
```

Comment:
This has turned up because these changes were committed to release/101, but the PR is against master.
We want these changes for release 101, so the PR should be against that release, which should then mean this bit goes away...